mininet/node.py

```diff
@@ -839,6 +839,42 @@ def init( cls ):
         cls.inited = True
 
 
+class XIAHost( Host ):
+    "eXpressive Internet Architecture host"
+
+    def __init__( self, name, **kwargs ):
+        Node.__init__( self, name, **kwargs )
+        self.xids = {}  # mapping of configurable inputs to XID types
+
+    def config( self, hid=None, **params):
+        """hid: private key file names for Host Identifiers eg. hid='pk1,pk2'
+           params: parameters for Node.config()"""
+        r = Node.config( self, **params )
+        self.setParam( r, 'setHID', hid=hid )
+        return r
+
+    def setHID( self, hid=None ):
+        "Assign HID(s) to the node."
+        if hid is None:
+            return
+        self.xids[ 'hid' ] = self.addParameters( hid )
```

Comment:
Once we make `hid` into a list, this line can just be:

    self.xids['hid'] = hid